lib/sync.js

```diff
@@ -360,21 +357,33 @@ SyncApi.prototype.sync = function() {
  */
 SyncApi.prototype.stop = function() {
     debuglog("SyncApi.stop");
+    if (global.document) {
+        global.document.removeEventListener("online", this._onOnline.bind(this), false);
```

Comment:
does this actually work? doesn't `.bind` give a different function to that which was originally bound?